### DISCUSSION TITLE
Please help /setup won't work

### DISCUSSION THREAD

**Discussion by FourInk** [Pryaxis/TShock#2365] [General]
I did something and now I can't do /setup anymore. It said to go here for help because I need owner on my account but I can't anymore! Please Help.

Edit: Thanks to the help of hakusaro the issue was solved! For anyone else having same problem you have to go to the server console and type "user group [your in-game name] owner".

**Comment by hakusaro** [comment]
Did you try the instructions on the pinned ["help! setup-code.txt is gone." thread](https://github.com/Pryaxis/TShock/discussions/2325)? If so, what option did you take?

**Reply by FourInk** [reply]
I tried both, the first just said "the user (name) already exists", and the second one I don't have access to the command.

**Reply by hakusaro** [reply]
You should run those commands from the server console if you can -- can you try from there instead?

**Reply by hakusaro** [reply]
Edit: I updated that answer to be a little more clear in terms of "run 1 & 2 of option 1, even if 1 says the user exists." You may need to reset the password on the first user, though.

**Reply by FourInk** [reply]
Ok so I tried from console and this time when I do /group no longer does it say "you do not have access to this permission" but I can't see the "(Owner)" beside my name anymore.

**Reply by hakusaro** [reply]
Well, once you connect to the server, are you logging into your newly changed account? You are, right?

**Reply by FourInk** [reply]
I guess

**Reply by FourInk** [reply]
Thanks for the help!

**Reply by hakusaro** [reply]
You guess? What are you doing when you connect? What do you see in chat?

**Reply by FourInk** [reply]
"Authenticated as Four succesfully"

**Reply by hakusaro** [reply]
Hmm, can you run commands in game? E.g., `/time day`?

**Reply by FourInk** [reply]
Yep!

**Reply by hakusaro** [reply]
Does `/accountinfo Four` say you're in the Owner group?
